boa/interpret.py

```diff
@@ -225,6 +227,42 @@ def load_partial(filename: str, compiler_args=None):
         )
 
 
+def loads_partial_vvm(source_code: str, version: str):
+    # will install the request version if not already installed
+    vvm.install_vyper(version=version)
+    compiled_src = vvm.compile_source(source_code)
+
+    abi = compiled_src["<stdin>"]["abi"]
+    bytecode = compiled_src["<stdin>"]["bytecode"]
+    # remove the 0x prefix
+    bytecode = bytes.fromhex(bytecode[2:])
```

Comment:
use `bytecode.removeprefix("0x")` as hex implementations are not always consistent